pkg/ecdsa/tss/protocol_recovery.go

```diff
@@ -14,6 +14,7 @@ import (
 // the case that a keep is terminated.
 func BroadcastRecoveryAddress(
 	parentCtx context.Context,
+	btcRecoveryAddress string,
```

Comment:
Our argument order is a mess here 😅

Reply:
What's the ordering convention?

Reply:
I'm mostly pointing out there is none 😭 we should figure something out but it'll probably take a while to play out.